File: myItem/qqrobot3/src/main/java/pers/fj/robot/FeiJiangRobotApplication.java
package pers.fj.robot;

import love.forte.simbot.spring.autoconfigure.EnableSimbot;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

/**
 * Author:feijiang
 * Date: 2022/9/19 14:23
 */
@EnableSimbot
@SpringBootApplication
public class FeiJiangRobotApplication {
    public static void main(String[] args) {
        SpringApplication.run(FeiJiangRobotApplication.class,args);
    }
}

File: myItem/qqrobot2/src/main/kotlin/pers/fj/robot/FeiJiangRobotApplication.kt
package pers.fj.robot

import love.forte.simboot.spring.autoconfigure.EnableSimbot
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

/**
 * Author:feijiang
 * Date: 2022/10/14 9:18
 */
@EnableSimbot
@SpringBootApplication
class FeiJiangRobotApplication
fun main(args: Array<String>){
    runApplication<FeiJiangRobotApplication>(*args)
}